### PR TITLE
Containerized Docker Application Lifecycle - 01.1 - Chapter 01 - Article 1 (bis)

### PR DESCRIPTION
**Pull request by mvelosop**
## Summary

Cherry pick from PR https://github.com/dotnet/docs/pull/9306 + fix title casing

This PR replaces https://github.com/dotnet/docs/pull/9306

### CODE REVIEW

**Comment by mvelosop** [comment]
BTW @mairaw, now the we finally worked through PR https://github.com/dotnet/docs/pull/9305 😅 I think I could:

1. Create all PRs at once, one per file (small 🙂).
2. Fix all contractions in a single PR by using Regex replacement, after all PRs from `#1` are merged.

Is that Ok for you?

**Comment by mairaw** [comment]
Sure. I'll go ahead and review this ignoring contractions then.

File: docs/standard/containerized-lifecycle-architecture/index.md [hunk 1]
```diff
@@ -7,23 +7,21 @@ ms.date: 11/23/2018
 ---
```
Comment (mairaw) [inline docs/standard/containerized-lifecycle-architecture/index.md]: change ms.date?

File: docs/standard/containerized-lifecycle-architecture/index.md [hunk 2]
```diff
@@ -7,23 +7,21 @@ ms.date: 11/23/2018
...
 
-Containers also isolate applications from one another on a shared operating system (OS). Containerized applications run on top of a container host, which in turn runs on the OS (Linux or Windows). Thus, containers have a significantly smaller footprint than virtual machine (VM) images.
+Containers also isolate applications from each other on a shared OS. Containerized applications run on top of a container host that in turn runs on the OS (Linux or Windows). Containers therefore have a much smaller footprint than virtual machine (VM) images.
 
-Each container can run an entire web application or a service, as shown in Figure 1-1.
+Each container can run a whole web application or a service, as shown in Figure 1-1. In this example, Docker host is a container host, and App1, App2, Svc1, and Svc2 are containerized applications or services.
 
-![](./media/image1.png)
+![Two applications and two services running on the OS in a VM or a physical server](./media/image1.png)
```
Comment (mairaw) [inline docs/standard/containerized-lifecycle-architecture/index.md]: another task that we'll have to do at a later time is to add meaningful file names for images including dashes to separate words. It impacts SEO.

**Review by mairaw — APPROVED** [review]
LGTM but you should change ms.date to reflect this was updated.

File: docs/standard/containerized-lifecycle-architecture/index.md (hunk 2 above)
Comment (mvelosop) [reply docs/standard/containerized-lifecycle-architecture/index.md]: Yep, I started doing it for new images, I didn't understand why at first, but now I'm using it even in my blogposts! 😊

This was one of the nice takeaways from this assignment (besides learning contractions, of course 😉).

Now I find myself asking quite often "How would Maira write this?" 😂

File: docs/standard/containerized-lifecycle-architecture/index.md (hunk 1 above)
Comment (mvelosop) [reply docs/standard/containerized-lifecycle-architecture/index.md]: Pushed another commit, although the date for the gross of the update was 11/23/2018!

Should I update all dates for the remaining articles? they all have 11/23/2018.

**Pull request merged**